specification/cognitiveservices/data-plane/AnomalyDetector/stable/v1.1/cadl/multivariate/routes.cadl

```diff
@@ -0,0 +1,130 @@
+@doc("""
+Create and train a multivariate anomaly detection model. The request must
+include a source parameter to indicate an externally accessible Azure blob
+storage URI.There are two types of data input: An URI pointed to an Azure blob
+storage folder which contains multiple CSV files, and each CSV file contains
+two columns, timestamp and variable. Another type of input is an URI pointed to
+a CSV file in Azure blob storage, which contains all the variables and a
+timestamp column.
+""")
+// To improve CADL implementation apply: https://github.com/Azure/azure-rest-api-specs/issues/21527
+op CreateAndTrainMultivariateModel(
+  @body body: ModelInfo,
+): {
+  @statusCode statusCode: 201,
+  @doc("Location and ID of the model.")
+  @header Location: string,
+  @body body: Model
+  } | ErrorResponse;
+
+@get
+@route("/multivariate/models")
+@summary("List Multivariate Models")
+@doc("List models of a resource.")
+// To improve CADL implementation apply: https://github.com/Azure/azure-rest-api-specs/issues/21527
+op ListMultivariateModels(
+  @doc("Skip indicates how many models will be skipped.")
+  @query
+  skip?: int32 = 0;
+
+  @doc("Top indicates how many models will be fetched.")
+  @query
+  top?: int32;
+): ModelList | ErrorResponse;
+
+
+@summary("Delete Multivariate Model")
+@doc("Delete an existing multivariate model according to the modelId")
+@delete
+@route("/multivariate/models/{modelId}")
+// To improve CADL implementation apply: https://github.com/Azure/azure-rest-api-specs/issues/21527
+op DeleteMultivariateModel(
+  @doc("Model identifier.")
+  @path
+  modelId: string,
+): {
+  @doc("Delete model successfully.")
+  @statusCode
+  statusCode: 204
+  } | ErrorResponse;
+
+@summary("Get Multivariate Model")
+@doc("""
+Get detailed information of multivariate model, including the training status
+and variables used in the model.
+""")
+@get
+@route("/multivariate/models/{modelId}")
+// To improve CADL implementation apply: https://github.com/Azure/azure-rest-api-specs/issues/21527
+op GetMultivariateModel(
+  @doc("Model identifier.")
+  @path
+  modelId: string,
+): Model | ErrorResponse;
+
+@route("/multivariate/models/{modelId}:detect-batch")
+@summary("Detect Multivariate Anomaly")
+@doc("""
+Submit multivariate anomaly detection task with the modelId of trained model
+and inference data, the input schema should be the same with the training
+request. The request will complete asynchronously and return a resultId to
+query the detection result.The request should be a source link to indicate an
+externally accessible Azure storage Uri, either pointed to an Azure blob
+storage folder, or pointed to a CSV file in Azure blob storage.
+""")
+// To improve CADL implementation apply: https://github.com/Azure/azure-rest-api-specs/issues/21527
+op DetectMultivariateBatchAnomaly(
+  @doc("Model identifier.")
+  @path
+  modelId: string,
+  @body body: DetectionRequest,
+): {
+  @statusCode statusCode: 202,
```

Comment:
This API is quite similar to LRO,  could you confirm this is not a LRO operation?

Reply:
There are indeed some LRO in AD, but they are long, so we decided at the last archboard to not generate LRO methods for now. This will be a topic for their GA though, but it gives us time.